index.html

```diff
@@ -3259,6 +3259,37 @@ <h2>A project for first-time contributions</h2>
         </div>
         <!-- ________ Contributor's card END ________  -->
 
+        
+         <!-- ________ Contributor's card START ________  -->
+        <div class="card">
+
+            <p class="name">Annapoorna</p>
+            <p class="contact">
+                <i class="fab fa-facebook-square"></i>
+                <a href="https://www.facebook.com/annapoorna.somayaji" target="_blank">@yourHandle</a>
+            </p>
+            <p class="about">
+                I like to learn to code with the help of online resources.
+            </p>
+            <div class="resources">
+                <p>3 Useful Dev Resources</p>
+                <ul>
+                    <li>
+                        <a href="https://www.w3schools.com/" target="_blank" title="This is a link to an awesome resource">Link to w3schools</a>
+                    </li>
+                    <li>
```

Comment:
In this one, if you are not adding an external link it's better if you delete the element from `<li>` to `</li>`. The same goes for the next link.